.github/CONTRIBUTING.md

```diff
@@ -108,9 +108,10 @@ The Branch Strategy is based on [git-flow-lite](http://nvie.com/posts/a-successf
 
 ### Coding styleguide
 
-This project follows the [Qt Coding Style](https://wiki.qt.io/Qt_Coding_Style). All submissions are expected to follow this style.
+The coding style of the project is enforced using llvm's `clang-format` formatting tool. A thorough description
+of the coding style can be found in the `.clang-format` file, but the main conventions are presented here.
```

Comment:
Would it be useful to mention here the use of `//clang-format off` and `//clang-format on` for having ClangFormat ignore certain sections of the code? To illustrate the possible use-cases, an example could also be provided. I'm thinking on something like:

When deemed necessary, you can use `//clang-format off` and `//clang-format on` to have ClangFormat ignore certain parts of the code. For instance, from project's code-base:

```cpp
// clang-format off
int binary =
    ((hmac[offset] & 0x7f) << 24)
    | ((hmac[offset + 1] & 0xff) << 16)
    | ((hmac[offset + 2] & 0xff) << 8)
    | (hmac[offset + 3] & 0xff);
// clang-format on
```

In the example above, the manual formatting resulted in more readable code than the ClangFormat changes, hence the need for using the special comment lines.